cranelift/docs/ir.md

```diff
@@ -405,7 +405,7 @@ retlist      : paramlist
 param        : type [paramext] [paramspecial]
 paramext     : "uext" | "sext"
 paramspecial : "sret" | "link" | "fp" | "csr" | "vmctx" | "sigid" | "stack_limit"
-callconv     : "fast" | "cold" | "system_v" | "fastcall" | "baldrdash_system_v" | "baldrdash_windows"
+callconv     : "fast" | "cold" | "system_v" | "fastcall"
```

Comment:
this misses the wasmtime variants of system_v and fastcall.

Reply:
Updated (and added the other missing ones too), thanks!